### ISSUE TITLE
Advertise to core contributors to take on more permissions rights & responsibilities

### ISSUE THREAD

**Issue by antoviaque** [openedx/community-wg#102]
This issue is to follow up on a topic from the contributor meetup working group

Need:
* Taking on more responsibilities (maintainer, repos reviews)
* Dedicate more time
* Obtain more rights

Potential solutions:
* Post dedicated thread/announcement, calling for:
	* Existing core contributors: get permissions on more repos, or groups of repos
	* Call for more people to join as core contributors & maintainers
	    * Remaing repos to assign: https://docs.google.com/spreadsheets/d/1rMwbRldYwp-xPsAG2vijqwB__9M7P2AgjDxX31G7ucA/edit?pli=1#gid=965363198
* We have an email list of core contributors - also email?
* Mention categories or repos we would like to assign: edx-platform (check list to be assigned)
	* Gather data on the repositories experiencing the longest delays in PR reviews and understand the reasons behind the delays (see ticket about OSPR monitoring)

See https://discuss.openedx.org/t/pull-requests-review-delays/10497 for the context, as well as [this discussion on a sprint update](https://discuss.openedx.org/t/contributors-meetup-async-update-september-16th-2023-september-29th-2023/11322/5?u=antoviaque)

**Comment by antoviaque** [comment]
From the [contributor meetup this week](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3856924673/2023-09-05+CC+Working+Group+Meeting+Notes) - the repositories looking for maintainers are listed on this page:

https://docs.google.com/spreadsheets/d/1rMwbRldYwp-xPsAG2vijqwB__9M7P2AgjDxX31G7ucA/edit?pli=1#gid=965363198

**Comment by jalondonot** [comment]
09/01 Contributor Meetup Update:
-  @e0d , @feanil, and @jmbowman  will schedule a kickoff session for the Maintenance WG work scope for 2024

**Comment by antoviaque** [comment]
It was also noted that it would be preferable to have a proper list of the work that needs to be done before doing the call for contributions, to have a concrete list to offer immediately.

**Comment by antoviaque** [comment]
@sarina Wrote and sent a nice email about this to the core contributors mailing list 👍 [https://groups.google.com/a/axim.org/d/msgid/core-contributors-dev/CAFtSF8y6V-4y3MqODcuSqGo6V1Z5h1PamwnVwYXUqtH%3Db2KOjw%40mail.gmail.com?utm_medium=email&utm_source=footer](https://groups.google.com/a/axim.org/d/msgid/core-contributors-dev/CAFtSF8y6V-4y3MqODcuSqGo6V1Z5h1PamwnVwYXUqtH%3Db2KOjw%40mail.gmail.com?utm_medium=email&utm_source=footer)

**Comment by antoviaque** [comment]
Two new initiatives:
* [Call for maintainers by TOC members](https://discuss.openedx.org/t/seeking-maintainers/12306/2) (Régis, Nacho & me)
* [Call for maintainers](https://discuss.openedx.org/t/introducing-the-maintenance-working-group/12159) by the [new maintenance working group](https://discuss.openedx.org/t/introducing-the-maintenance-working-group/12159)

**Comment by antoviaque** [comment]
Another initiative yesterday with a meeting calling Open edX partners to focus contributions on maintenance. Part of the requirements to be an Open edX partner is to contribute to the project's core contributor program, which can include maintenance of the project.

https://otter.ai/u/QyuMZ_sLsvShWD36ahfVkLGQgr8?view=summary&tab=chat

**Comment by angonz** [comment]
Hi Xavier, we are a very small community, and we need to grow to get more volunteers and opportunities to contribute. In my opinion, adding more requirements to become a partner will only set additional entrance barriers and drive off newcomers. We'd better think of incentives to encourage partners and partner candidates to engage in core contributions.

**Comment by e0d** [comment]
@angonz we do need to grow, I agree.  However, this is not a new requirement.  Rather we are asking partners to use the time they have committed to contribute specifically for maintenance work which benefits everyone.

**Comment by antoviaque** [comment]
+1, contributing core contributor work is an important part of what makes it a partnership - it's important to find a way to share this maintenance work. It's not too much work if we all do it together, adapted to the size of each organization - but it can be quite a lot for a single entity to carry entirely on its own, as we have seen with 2U.

Also becoming a maintainer of the project is a bit like owning a piece of it - it becomes ours, and this is important to build a strong community. It will grow the voice of the community and its influence on decisions, in the mid and long term.

That said @angonz for growth we can also create intermediary statuses, with something lighter than "partner"? This way those who are just getting their feet wet into contributing can have smaller goals and steps. And bigger commitments are reserved for top-tier later statuses, like partner.

**Comment by antoviaque** [comment]
During the last core contributor meetup, @jalondonot remarked that it would be useful to do a new call for maintainers within the larger community. It could be useful to provide a list of specific repositories we are currently looking for maintainers. 

There is the canonical list of priorities at https://docs.google.com/spreadsheets/d/1rMwbRldYwp-xPsAG2vijqwB__9M7P2AgjDxX31G7ucA/edit?pli=1&gid=0#gid=0 -- but it might be a bit hard to parse for someone not deeply involved in the maintainers working group. 

@feanil Are there specific repos from that list which we should point to? Any that doesn't have a maintainer listed now, by priority order? Should we exclude the ones for which there is owner interest listed, or not?

**Comment by feanil** [comment]
@antoviaque I've created a new view of the spreadsheet to make this easier to see hopefully, but yes you're on the right track, I removed any repos that have maintainers or are on track to have a maintainer and the rest of the list sholud be viewable in priority order to figure out what we deem most important first.

New view: https://docs.google.com/spreadsheets/d/1rMwbRldYwp-xPsAG2vijqwB__9M7P2AgjDxX31G7ucA/edit?pli=1&gid=0#gid=0&fvid=453172719

**Comment by antoviaque** [comment]
@feanil Thank you!

@ehuthmacher Would it be possible to do a blog post about this, to call everyone in the community to contribute maintenance to a list of specific repos we would list? Who would be the best person to write that blog post? Maybe @e0d as both head of the TOC and member of Axim, or you @feanil as chair of the maintenance WG & Axim? As a fallback, I could also write it.

**Comment by feanil** [comment]
Sure, I'll put something in my schedule to do this next week.

**Comment by antoviaque** [comment]
Blog post draft being reviewed at https://docs.google.com/document/d/11wommeGRnVIXV1CJgy7CCALIQ2P5m5d11RqKMdH79Gc/edit

**Comment by antoviaque** [comment]
@feanil @ehuthmacher  Btw if you need help with any of the steps, like the formatting/publishing of the blog post, or anything else, don't hesitate, I would be happy to help.

**Comment by ehuthmacher** [comment]
Thank you @antoviaque . The blog has been published[ here. ](https://openedx.org/blog/seeking-more-maintainers/)
